src/client/components/features/Alternance/Rechercher/FormulaireRecherche/FormulaireRechercheAlternance.test.tsx

```diff
@@ -207,4 +207,33 @@ describe('FormulaireRechercheAlternance', () => {
 		const rayon = screen.getByTestId('Select-InputHidden');
 		expect(rayon).toHaveValue('10');
 	});
+
+	it('laisse le champ domaine vide quand il manque les codes romes dans les query params', () => {
+		mockUseRouter({ query: {
+			libelleMetier: 'Boulangerie, pâtisserie, chocolaterie',
+		} });
+
+		render(
+			<DependenciesProvider metierService={aMetierService()} localisationService={aLocalisationService()}>
+				<FormulaireRechercheAlternance />
+			</DependenciesProvider>,
+		);
+
+		const inputMétiers = screen.getByRole('combobox', { name: 'Domaine' });
+		expect(inputMétiers).toHaveValue('');
+	});
+	it('laisse le champ domaine vide quand il manque le libellé dans les query params', () => {
+		mockUseRouter({ query: {
+			codeRomes: 'D1102,D1104',
+		} });
+
+		render(
+			<DependenciesProvider metierService={aMetierService()} localisationService={aLocalisationService()}>
+				<FormulaireRechercheAlternance />
+			</DependenciesProvider>,
+		);
+
+		const form = screen.getByRole('form');
+		expect(form).not.toHaveFormValues({ codeRomes: expect.anything() });
```

Comment:
J'ai du mal a comprendre le lien entre le 'it' et ce que tu testes vraiment. De ce que je comprends de l'implem du test:  si on n'a pas de libellé dans les query params mais on a des codes romes, on verifie qu'il y a pas de codesRomes dans les valeurs du formulaire du coup c'est pas vraiment 'laisse le champ domaine vide quand il manque le libellé dans les query params' surtout vis à vis du test au dessus